README.md

```diff
@@ -9,16 +9,54 @@ https://zookeeper.ibinx.com/master/
 
 ### Requirements
 
-* PHP 5.6 or later with MySQL PDO driver
-* MySQL/MariaDB
-
+* Apache 2.4
+* MySQL/MariaDB 5.6+
+* PHP 7.2 & PHP 7.2-mysql (thread safe version required)
 
 ### Getting Started
 
-1. Clone the repository;
+
+The PHP setup can be installed either by manually installing its constituent
+components or by installing and integrated 'XAMPP' framework. The procedure
+using the latter is as follows:
+
+  - cd <SRC_DIR>
+  - git clone https://github.com/RocketMan/zookeeper.git
+  - git checkout -b kzsu
```

Comment:
The instruction "git checkout -b kzsu" is specific to kzsu.  It is OK to include this if you qualify it to say something like this:  "If you want to work on a branch other than master, then check it out now.  For example, to work on the 'kzsu' branch:  git checkout -b kzsu "